src/Tests/Microsoft.NET.Build.Tests/GivenThatWeWantToControlGeneratedAssemblyInfo.cs

```diff
@@ -89,6 +91,175 @@ public void It_respects_opt_outs(string attributeToOptOut)
+            {
+                Name = "ProjectWithSourceRevisionId",
+                IsSdkProject = true,
+                TargetFrameworks = "netcoreapp2.0",
+            };
+
+            var testAsset = _testAssetsManager.CreateTestProject(testProject)
+                .WithProjectChanges((path, project) =>
+                {
+                    var ns = project.Root.Name.Namespace;
+
+                    project.Root.Add(
+                        new XElement(ns + "Target",
+                            new XAttribute("Name", "InitializeSourceControlInformation"),
+                            new XElement(ns + "PropertyGroup",
+                                new XElement("SourceRevisionId", "xyz"))));
+
+                    project.Root.Add(
+                        new XElement(ns + "PropertyGroup",
+                            new XElement("SourceControlInformationFeatureSupported", "true"),
+                            new XElement("IncludeSourceRevisionInInformationalVersion", "false")));
+                });
+
+            testAsset.Restore(Log, testProject.Name);
+
+            var command = new GetValuesCommand(Log, Path.Combine(testAsset.TestRoot, testProject.Name), "netcoreapp2.0", valueName: "InformationalVersion");
+            command.Execute().Should().Pass();
+
+            command.GetValues().ShouldBeEquivalentTo(new[] { "1.0.0" });
+        }
+
+        [Fact]
+        public void It_includes_source_revision_id_if_available__version_without_plus()
+        {
+            TestProject testProject = new TestProject()
+            {
+                Name = "ProjectWithSourceRevisionId",
+                IsSdkProject = true,
+                TargetFrameworks = "netcoreapp2.0",
+            };
+
+            var testAsset = _testAssetsManager.CreateTestProject(testProject)
+                .WithProjectChanges((path, project) =>
+                {
+                    var ns = project.Root.Name.Namespace;
+
+                    project.Root.Add(
+                        new XElement(ns + "Target",
+                            new XAttribute("Name", "_SetSourceRevisionId"),
+                            new XAttribute("BeforeTargets", "InitializeSourceControlInformation"),
+                            new XElement(ns + "PropertyGroup",
+                                new XElement("SourceRevisionId", "xyz"))));
+
+                    project.Root.Add(
+                        new XElement(ns + "Target",
+                            new XAttribute("Name", "InitializeSourceControlInformation")));
+
+                    project.Root.Add(
+                        new XElement(ns + "PropertyGroup",
+                            new XElement("SourceControlInformationFeatureSupported", "true")));
+                });
+
+            testAsset.Restore(Log, testProject.Name);
+
+            var command = new GetValuesCommand(Log, Path.Combine(testAsset.TestRoot, testProject.Name), "netcoreapp2.0", valueName: "InformationalVersion");
```

Comment:
Use `testProject.TargetFrameworks` instead of hardcoding `netcoreapp2.0` as the target framework to pass to the GetValuesCommand method (here and elsewhere).

Reply:
OK.